priv/base/env.sh

```diff
@@ -238,8 +238,8 @@ check_user() {
         #     riak-admin bucket-type create mytype {props: {n_val: 4}}
         #  after the arguments were passed into the new shell during exec su
         #
-        # So this regex finds any '"', '{', or '}' and prepends with a '\'
-        ESCAPED_ARGS=`echo "$@" | sed -e 's/\([{}"]\)/\\\\\1/g'`
+        # So this regex finds any '(', ')', "'" , '"', '{', or '}' and prepends with a '\'
+        ESCAPED_ARGS=$(echo "$@" | sed -e "s/\([\\\(\\\){}\"']\)/\\\\\1/g")
```

Comment:
```ESCAPED_ARGS=`echo "$@" | sed -e 's/\([\\\(\\\){}"\x27]\)/\\\\\1/g'` ``` should be equivalent.

The default Solaris shell only supports command substitution with backticks. The single quote character `'` can be expressed in sed expressions as `\x27`.